scripts/dependencies/lima.ts

```diff
@@ -50,6 +58,31 @@ export class LimaAndQemu implements Dependency, GithubDependency {
         }
       });
     });
+
+    // Download and install forward compatible limactl binary built for the newest Darwin versions
+    if (platform === 'macos') {
```

Comment:
For M1 builds platform is now `macos-aarch64`:

```suggestion
    if (platform.startsWith('macos')) {
```